dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/CommunityManagerTest.kt

```diff
@@ -36,10 +36,6 @@ class CommunityManagerTest {
     private val apiAccessor = ApiAccessor()
     private val requestControllerApi = RequestControllerApi(BASE_PATH_TO_COMMUNITY_MANAGER)
 
-    private fun authenticateAsTechnicalUser(technicalUser: TechnicalUser) {
```

Comment:
ok that is the function which was moved to the apiacessor. Should it then also be consistently used in the apiAcessor?

Reply:
i removed it, since it doesn't add anything

Reply:
ok